development/decision_process.rst

```diff
@@ -0,0 +1,182 @@
+######################################
+Discussion and Decision Making Process
+######################################
+
+This page describes how decisions are made in Data Management.
+Generally, DM teams and staff have agency to implement non-controversial work (see § :ref:`decision-making-empowerment`).
+
+For cases where you have a well-formed proposal that requires project consensus we use a :ref:`Request for Comments (RFC) <decision-making-rfc>` process.
```

Comment:
I'm not sure "requires project consensus" is correct, and I'm not sure that it's an obvious criterion, as opposed to the original's A/B/C three-part test.